File: generated_code_examples/visual_basic/regression/xgboost.vb
Module Model
Function Score(ByRef inputVector() As Double) As Double
    Dim var0 As Double
    If inputVector(12) >= 9.725 Then
        If inputVector(12) >= 19.23 Then
            var0 = 3.5343752
        Else
            var0 = 5.5722494
        End If
    Else
        If inputVector(5) >= 6.941 Then
            var0 = 11.1947155
        Else
            var0 = 7.4582143
        End If
    End If
    Dim var1 As Double
    If inputVector(12) >= 5.1549997 Then
        If inputVector(12) >= 15.0 Then
            var1 = 2.8350503
        Else
            var1 = 4.8024607
        End If
    Else
        If inputVector(5) >= 7.406 Then
            var1 = 10.0011215
        Else
            var1 = 6.787523
        End If
    End If
    Score = 0.5 + (var0 + var1)
End Function
End Module

File: generated_code_examples/c_sharp/regression/xgboost.cs
namespace ML {
    public static class Model {
        public static double Score(double[] input) {
            double var0;
            if ((input[12]) >= (9.72500038)) {
                if ((input[12]) >= (16.0849991)) {
                    var0 = 1.36374998;
                } else {
                    var0 = 1.96335626;
                }
            } else {
                if ((input[5]) >= (6.94099998)) {
                    var0 = 3.74704218;
                } else {
                    var0 = 2.48376822;
                }
            }
            double var1;
            if ((input[12]) >= (9.63000011)) {
                if ((input[12]) >= (19.2299995)) {
                    var1 = 1.05852115;
                } else {
                    var1 = 1.68638802;
                }
            } else {
                if ((input[5]) >= (7.43700027)) {
                    var1 = 3.95318961;
                } else {
                    var1 = 2.40278864;
                }
            }
            return ((0.5) + (var0)) + (var1);
        }
    }
}
